packages/nodejs-lts/build.sh

```diff
@@ -16,6 +16,9 @@ TERMUX_PKG_SUGGESTS="clang, make, pkg-config, python"
 TERMUX_PKG_RM_AFTER_INSTALL="lib/node_modules/npm/html lib/node_modules/npm/make.bat share/systemtap lib/dtrace"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_HOSTBUILD=true
+if [ $TERMUX_PACKAGE_FORMAT = "pacman" ]; then
+	TERMUX_PKG_PROVIDES="nodejs"
```

Comment:
From your previous PR's requested changes by @xeffyr 

> At max you can specify TERMUX_PKG_PROVIDES="nodejs" for package nodejs-lts for both pacman and debian package types.

So, would you like to do this for both Debian and Pacman format